python/addBTV.py

```diff
@@ -363,6 +449,7 @@ def add_BTV(process, runOnMC=False, onlyAK4=False, onlyAK8=False, keepInputs=['D
         extension=cms.bool(True),  # this is the extension table for FatJets
         variables=cms.PSet(
             CommonVars,
+						SubJetVars,
```

Comment:
[Style] Indentation level seems to vary for the newly added lines, compared to the surrounding statements